cpg_workflows/large_cohort/combiner.py

```diff
@@ -26,7 +26,8 @@ def _check_gvcfs(sequencing_groups: list[SequencingGroup]) -> list[SequencingGro
                     f'Sequencing group {sequencing_group} is missing GVCF. '
                     f'Use workflow/skip_sgs = [] or '
```

Comment:
I think this argument will be disappearing soon following the implementation of custom cohorts, @vivbak ?